### ISSUE TITLE
Ubuntu Servers Running

### ISSUE THREAD

**Issue by frankiexile** [cryptonotefoundation/cryptonote#261]
How do I keep the 2 Ubuntu seeds running the coin so that people can use the wallets/mine even when my computer is off? Can't connect to it when they're off so I was wondering what all of the cryptos like bytecoin, monero, etc use to keep the coin running 24/7

**Comment by SoraKohaku** [comment]
vps lol buy vps for this~

**Comment by frankiexile** [comment]
I know that I mean I have the 2 Ubuntu VPS but whenever I close out of the VPS prompts after starting ./coind the service shuts down. How can I have the coin running 24/7 even when I close out of the VPS prompts. I'm using puTTy to ssh into the vps

**Comment by frankiexile** [comment]
I think i read somewhere that this is called an upstart.. how would I manage to have upstart for cryptonote

**Comment by frankiexile** [comment]
Also just read that upstart doesn't work in Ubuntu 16.04 so it is now systemd. Any idea how to use this feature for keeping the seeds running automatically forever?

**Comment by SoraKohaku** [comment]
im used 2 debian for this. jessie is good~

**Comment by jdmedeiros** [comment]
`nohup ./your_coin >/dev/null 2>&1 &`

**Comment by SoraKohaku** [comment]
why not used ./daemon --testnet before try alive?

**Comment by frankiexile** [comment]
I used systemd enable for the ubuntu servers to start up ./coind. is this good ?

**Comment by frankiexile** [comment]
[Unit]
Description=Daemon

[Service]
Type=simple
ExecStart=/root/coin/build/release/src/coind --no-console
Restart=on-failure
RestartSec=30

[Install]
WantedBy=multi-user.target

**Comment by SoraKohaku** [comment]
yes good but you can try testnet before it. this wanna used CN-7. or you have mission to real project?

**Comment by SoraKohaku** [comment]
same as usual i think
